src/utils/utils.ts

```diff
@@ -139,12 +139,10 @@ export module _Util {
      * @param {number} count The length of the array to generate
      * @return {any[]}
      */
-    export function createFilledArray<T>(value: T, count: number): T[];
-    export function createFilledArray<T>(func: (index?: number) => T, count: number): T[];
-    export function createFilledArray<T>(value: any, count: number) {
+    export function createFilledArray<T>(value: T | ((index?: number) => T), count: number) {
```

Comment:
please update above TSDoc to match (I can't comment on the actual TSDoc line for some reason).